cli/src/installers/drizzle.ts

```diff
@@ -55,6 +55,7 @@ export const drizzleInstaller: Installer = ({
   packageJsonContent.scripts = {
     ...packageJsonContent.scripts,
     "db:push": "dotenv drizzle-kit push:mysql",
+    "db:studio": "dotenv drizzle-kit studio",
```

Comment:
running regular `pnpm drizzle-kit studio` causes the same env issue as we had with push before

Reply:
👍👍

Reply:
done